### ISSUE TITLE
These content after BottomSheet still can be clicked

### ISSUE THREAD

**Issue by xtuzy** [naweed/Maui.Controls.BottomSheet#1]
Current:

https://user-images.githubusercontent.com/17793881/186628120-d288cbdc-24e6-4cbc-a928-508fbb6281e5.mp4

Hope like this:

https://user-images.githubusercontent.com/17793881/186629864-91344dea-f8f4-4a3c-9db5-cbf7db9a01f0.mp4

**Comment by naweed** [comment]
Yes, I have this issue on almost everything on Maui.Android. On iOS, this issue is not there. Hopefully, this will be fixed by MAUI team soon.

**Comment by Buryyy** [comment]
As a temporary hack, this issue can be avoided by having an empty TagGestureReconizer on the base Grid of the content view.

```
            <Grid.GestureRecognizers>
                <TapGestureRecognizer Command="{x:Null}" />
            </Grid.GestureRecognizers>
```

**Comment by nellydcp** [comment]
I'm currently checking this project to use it in my personal project and I followed Buryyy's suggestion but with the following change:
`<Grid.GestureRecognizers>
        <TapGestureRecognizer Tapped="CloseBottomSheetButton_Tapped"/> 
</Grid.GestureRecognizers>`

This way, the sheet will disappear whenever I click outside of it.

**Comment by langhaoabcd** [comment]
```
<Grid.GestureRecognizers>
            <SwipeGestureRecognizer Direction="Down" Swiped="CloseBottomSheetButton_Swiped"/>
            <TapGestureRecognizer  Command="{x:Null}" Tapped="CloseBottomSheetButton_Tapped"/>
</Grid.GestureRecognizers>
```
